### DISCUSSION TITLE
Prometheus RAM usage on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by slaecker** [carlosedp/cluster-monitoring#137] [Q&A] [answered]
How can people happily use Prometheus to monitor their Raspberry Pi Kubernetes clusters and write about it on the web if it eats up all available RAM within minutes and brings a whole cluster node down. At least that's what happens to me whenever I spin up the prometheus stack, be it kube-prometheus-stack helm chart or this cluster-monitoring repo. As soon as the prometheus pod starts working it fills up the nodes 4 GB RAM within a few minutes.

Is there anything I'm missing? Any special settings?

How are you guys running this beast?

**Comment by carlosedp** [comment, marked as answer]
No special settings, it should work but depends on the size of the cluster and amount of applications.
